modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -257,10 +263,11 @@ It is recommended that each {product-title} node in the cluster must have access
 [discrete]
 [id="installation-vsphere-installer-infra-requirements-_{context}"]
 === Required IP Addresses
-An installer-provisioned vSphere installation requires two static IP addresses:
+An installer-provisioned vSphere installation requires these static IP addresses:
 
 * The **API** address is used to access the cluster API.
 * The **Ingress** address is used for cluster ingress traffic.
+* The control plane node addresses are used when upgrading a cluster from version 4.5 to 4.6.
```

Comment:
Unclear here why "API" and "Ingress" are bold - are they field labels in a GUI or something? If so, is "control plane node" also a label? It is possible that the bold should not be consistent on these three items, but want to verify that makes sense.

Reply:
Thanks, @jeana-redhat . I removed the bold from "API" and "Ingress."